libs/cdk/tsconfig.lib.json

```diff
@@ -8,7 +8,8 @@
     "lib": ["dom", "es2018"]
   },
   "angularCompilerOptions": {
-    "enableIvy": false,
+    "enableIvy": true,
```

Comment:
`enableIvy` appears to have been removed as a compiler option on the latest versions of Angular.

https://github.com/angular/angular/commit/4ed8dc858aebfdf0ab271ec8c710e60f3d10058a#diff-0a65f1d37c6a41d9b8c6e91e484e9e58bb815f22e9b3425432c086c1b904eefa


The nx migration screen to 13.2 automatically removes this option too.

Reply:
We also have to remove umd options like `umdModuleIds`.

Reply:
Nevermind, it's already done :+1: